eventdiscovery-sdk/src/main/res/values/theme.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <!-- This sets the default theme to light -->
+    <style name="SchedJoules_Theme.Default" parent="SchedJoules_Theme.Light"/>
```

Comment:
I don't agree with his reasoning. If I inherit from `ThemeA.B` I don't care if `ThemeA` is the actual parent of `ThemeA.B` or not. I want to inherit from `ThemeA.B` and not from `ThemeA`.

FWIW, the support library uses the dot notation with explicit parenting for pretty much every style. 

Also, for our partners it doesn't matter at all, because they should only use `SchedJoules_Theme.Default`, `SchedJoules_Theme.Light` or `SchedJoules_Theme.Dark` and just as outlined above, they don't care if these themes inherit from `SchedJoules_Theme` or not.
Everything else is internal business and I think our themes are still simple enough to not get it wrong.